content/docs/user-guide/gems/reference/aws/aws-gamelift/cpp-api.md

```diff
@@ -5,15 +5,15 @@ description: "Learn how to use the C++ API for AWS GameLift Gem in O3DE"
 toc: true
 ---
 
-## 1. Session Interface
+## Session Interface
 
 The *session interface*, `ISessionRequests` and `ISessionAsyncRequests`, provides public APIs that allows you to create game sessions and allows players to search and join online games. The session interface abstracts the implementation details of session related management.
```

Comment:
```suggestion
The *session interface*, `ISessionRequests` and `ISessionAsyncRequests`, provides public APIs that allow you to create game sessions and allows players to search and join online games. The session interface abstracts the implementation details of session-related management.
```
* As written, it's unclear if "*session interface*, `ISessionRequests` and `ISessionAsyncRequests`," in the first sentence is a three-item series that needs a comma after "`|SessionRequests`", OR if "*session interface*" is comprised of "`ISessionRequests` and `ISessionAsyncRequests`". (If the latter, set aside with an em dash.)
* Fixed a couple of subject-verb disagreement errors in this sentence ("...APIs that allows you to...and allows players to....")
  * If the phrase mentioned before is meant to be a three-item series, then there's another of these errors. ("The [a, b, and c] provides...")
* Hyphenated "session related"